agent/xds/listeners.go

```diff
@@ -401,7 +401,7 @@ func makeListenerFromUserConfig(configJSON string) (*envoy.Listener, error) {
 // specify custom listener params in config but still get our certs delivered
 // dynamically and intentions enforced without coming up with some complicated
 // templating/merging solution.
-func injectConnectFilters(cfgSnap *proxycfg.ConfigSnapshot, token string, listener *envoy.Listener, setTLS bool) error {
```

Comment:
This was added in 137a2c32c6e7ca6ca475f50be8a8c97532fa06ad (#7671) but never used.

@freddygv does this fix look correct, or should it be used somewhere? I don't know this code well enough to be confident in my change.

Reply:
That is safe to remove. Got missed during cleanup when wrapping up that PR.